### ISSUE TITLE
[1.12.2] Dimension ID Conflict: Oberon tries to override Galacticraft core´s Moon planet while landing. (Comes again at Server Side)

### ISSUE THREAD

**Issue by FrostB0rn** [MJRLegends/ExtraPlanets#388]
### ---Issue Report---
**(PLEASE DONT DELETE THIS TEMPLATE, YOUR ISSUE WILL BE IGNORED AND CLOSED IF YOU DO.)**

**Have you checked Closed Issues (if applicable & PLEASE CHECK BEFORE POSTING):** Yes

### Description of Issue
**This post is similar to the RoseQuartz-z or DartGaming ones. And It came the same issue again and as a Rose had said, Moon with Oberon.**

This bug consists that when you select the Moon at the galacticraft planet at server side, seems that GalacticCraft confuses the Moon seed with the Jupiter´s Moon, "Oberon"...

First thing which happens, you get stuck at GC loading screen.
At the server log will put as this

_"[Server thread/INFO] [Galacticraft]: Found matching world (-3980) for name: moon.oberon"_

And why I keep stuck at my GC loading screen?
Because as your mod ExtraPlanets is forcing Galacticraft to go Oberon while I have ordered GC to go to the Moon instead and due both have the same value and overrides themselves... here borns a "lock" in the client screen, causing a fail while removing the loading screen.
And the game is still on... Did you hear a explosion in the video evidence and still at the loading screen? Thats the rocket crashing into the oberon planet. The screen is still stuck until I relog into the server!

Tried to kick myself on the server side but didnt run, I still kept stuck.

After that, when I have reopened the client and go to my server, welp...
You will remain in Oberon with some radiation problems even we havent discovered the planet.

I found this after re-log on the server and before suddenly appear on Oberon

_"[15:48:08] [Server thread/INFO] [Galacticraft]: Server attempting to transfer player FrostBornMC to dimension -3980"_
(I had the Moon seed at default (-29) and that log doesnt show that seed is the Moon Im pointing for...)

**_I tried to change the seed of Oberon (as showed before), I tried again but it kept happening the same.
The evidence is right, I changed the seed from his default value -1506 to -3980_**

Take you an evidence of the bugs action below.
I performed it in creative for dont be dead at the action.

**_And this happens all time._**

### Steps to Reproduce Issue
_1. Install Minecraft forge Ver 14.23.5.2506 both client & server side
2. Install Galacticraft Core and Planets Build x.xx.x.187 both for 1.12.2 both client & server side
3. Install MJRLegendsLib & Extra Planets for 1.12.2 both client & server side
4. Install Planet Progression for 1.12.2 for 1.12.2 both client & server side (You will see on the video Oberon isnt discovered yet, my godness.)
5. Create & launch the forge server, then your client.
7. Enter to your server & launch yourself into a rocket. (In server takes a while loading the planets for first time)
8. Double-click the earth to zoom-in.
9. Select the moon and click "Launch"
10. Crash appears, you get stuck and may appear an explosion due Rocket crashed.
11. Use your taskbar to close the game and re-open it. Then, try relog to the server.
12. You suddenly appear on Oberon._

**Crash log/Log (if applicable):** [PASTEBIN. Do NOT copy and paste the log into the issue post itself.]
> *How do i locate a crash log on Minecraft on Curse/Twitch? see* https://gyazo.com/3c616a7dfcbe69033ea210805b093f82
> *How do i locate a crash log on Minecraft on FTB? see* http://imgur.com/a/IbWNC

**Screenshots (if applicable):** 

**Version of Mod using (PLEASE DONT USE 'Latest' OR 'Newest'):** 0.4.6 for Minecraft 1.12.2

**Where did this bug happen:** [Single Player or Server]
Server side.

### Additional Information
Can I send you a youtube video as an evidence?
This explains the process.
https://youtu.be/oxGD_8ICCoU

**Comment by MJRLegends** [comment]
A dimension id conflict isnt a bug. You just need the change them via the config

**Comment by FrostB0rn** [comment]
Are you meaning to client or server?

If you are meaning to server
I had changed that seed and still happened the same...

**Comment by MJRLegends** [comment]
You mention a crash but no crash log, thats the really issue here

**Comment by FrostB0rn** [comment]
I know... then I was wrong... I was meaning about a bug. This happened me today while playing in our private server. I tried to change the config on the servers config at "ExtraPlanets.cfg" into the dimensions settings and that "bug" was still happening.
Sorry If I got you confused...

**Comment by MJRLegends** [comment]
Right, please remove everything but Galacticraft, ExtraPlanets, MJRLegendsLib and test. If it works then this bug report is void and should be on a different mods issue tracker

**Comment by FrostB0rn** [comment]
Oh, dont tell me this is the "Crash-reports" section.

I was thinking it was for all issues.

**Comment by MJRLegends** [comment]
Please see my above message

**Comment by FrostB0rn** [comment]
Then this is void. :/

Anyway, I will test it again, thanks.

**Comment by FrostB0rn** [comment]
Ok, I will check it.

**Comment by MJRLegends** [comment]
Please if it does work with just Galacticraft, ExtraPlanets, MJRLegendsLib, try with Planet Progressions added. If that causes the issue then this is on the wrong issue tracker. If adding that mod doesnt cause issues. then you will need to keep adding mods back in till the issue starts and report it to them

**Comment by MJRLegends** [comment]
Ok please answer the following

- Does Galacticraft, ExtraPlanets, MJRLegendsLib have the issue?

- Does Galacticraft, ExtraPlanets, MJRLegendsLib, Planet Progressions have the issue?

**Comment by FrostB0rn** [comment]
- ExtraPlanets & Galacticraft have really the issue. Any of 95 mods put on my modpack had issues with your mod.
- The same as above.

**Comment by MJRLegends** [comment]
Lets try again. Ok please answer the following

When Galacticraft, ExtraPlanets, MJRLegendsLib are installed is there a issue?

When Galacticraft, ExtraPlanets, MJRLegendsLib, Planet Progressions are installed is there a issue?

**Comment by FrostB0rn** [comment]
- There isnt any "issue" at server and at client. The only issue i had was what I have said at the top.
- As same as above.

**Comment by MJRLegends** [comment]
You've confused me

**Comment by FrostB0rn** [comment]
Me too. I thought this was a place to comment any type of issue including bugs at your mods...

I havent found any place to report it unless here :/

**Comment by FrostB0rn** [comment]
Well, the issue has ExtraPlanets who causes this conflict with Galactic, first of all. The issue is about Extra Planets as this issue tracker is.

**Comment by MJRLegends** [comment]
This is the place to report ExtraPlanets issues yes, congrats you have found it.

You seem to giving me mixed answers, and your sentences dont make sense 

> There isnt any "issue" at server and at client which makes it crash
 means nothing its just words put together

**Comment by MJRLegends** [comment]
Do you have discord?

**Comment by FrostB0rn** [comment]
Of course.

**Comment by MJRLegends** [comment]
Please join my voice chat on my discord, https://discord.gg/SBnvQHc
